Smartfridge Fix

GhostActual opened the pull request:
Fixes the Hydroponics fridge issue that players were having problems with.

Also adds another airlock to the south of the surface-3 hangar because I felt that it didn't quite make sense for a hangar to only have one airlock and the rest be open to the elements.

GhostActual commented:
https://github.com/VOREStation/VOREStation/pull/11243 This PR fixes this issue

Arokha commented:
So they will have two produce smartfridges then?
![image](https://user-images.githubusercontent.com/15028025/128614808-758489da-8e37-44cb-95cd-5bfe8edd53e0.png)

GhostActual commented:
> So they will have two persistent produce smartfridges then? I think that's not desired and would require some balancey talks maybe. Not sure though.
> 
> ![image](https://user-images.githubusercontent.com/15028025/128614808-758489da-8e37-44cb-95cd-5bfe8edd53e0.png)

That's not the map I changed, I also added a second chicken, you're looking at an older file it seems.

Very-Soft commented:
> > So they will have two persistent produce smartfridges then? I think that's not desired and would require some balancey talks maybe. Not sure though.
> > ![image](https://user-images.githubusercontent.com/15028025/128614808-758489da-8e37-44cb-95cd-5bfe8edd53e0.png)
> 
> That's not the map I changed, I also added a second chicken, you're looking at an older file it seems.

it's a small thing, but they don't need another chicken, they can get more chickens through eggs or through cargo. Pls no chicken powercreep

Arokha commented on maps/tether/tether-03-surface3.dmm:
```diff
@@ -29854,7 +29844,7 @@
 /obj/effect/floor_decal/corner/grey/diagonal,
 /obj/machinery/door/firedoor/glass,
-/obj/machinery/smartfridge/produce,
+/obj/machinery/smartfridge/produce/persistent_lossy,
```
Yes, this is the map you changed. What are you talking about???

Huenererschrecker commented:
Since I've done the original pr and all that, the original idea was to just move the persistency fridge into the position between botany and the kitchen and delete the one in the backroom. The reason being that the one in the backroom is never used anyways. The plan was never to be 2 persistency fridges.

Arokha commented:
> Since I've done the original pr and all that, the original idea was to just move the persistency fridge into the position between botany and the kitchen and delete the one in the backroom. The reason being that the one in the backroom is never used anyways. The plan was never to be 2 persistency fridges.

I'll note that moving them erases their contents, just FYI.
